release.yaml

```diff
@@ -22,6 +22,7 @@ rancher-istio:
 - 1.10.400
 - 100.1.1+up1.11.4
 - 100.1.2+up1.11.4
+- 100.1.3+up1.11.7
```

Comment:
The last released version in `release-v2.6` is `100.1.0+up1.11.4` as seen in https://github.com/rancher/charts/tree/release-v2.6/charts/rancher-istio.

Therefore, in this release we should be releasing `100.1.1+up1.11.7`, based on the versioning scheme outlined in https://github.com/rancher/charts#versioning-charts.

This also means we need to remove [100.1.1+up1.11.4](https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-istio/100.1.1%2Bup1.11.4) and [100.1.2+up1.11.4](https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-istio/100.1.2%2Bup1.11.4) from `dev-v2.6`.

We also need to modify the release.yaml accordingly and move it into the commit **before** make charts, since that's how we standardized on making changes (manual changes first, automated changes such as make charts come in the end).

Can you make the appropriate changes in this PR? e.g. 
- Undo the `make charts` commit entirely
- Modify commit 2 to have the istio `package.yaml` to go back to `100.1.1+up1.11.7`
- Merge in the current commit 4 into commit 2 and fix the release.yaml to add `100.1.1+up1.11.7` and remove the other Istio entries (except for 1.10.400, which came from forward porting a chart in release-v2.5)
- run `CHART=rancher-istio VERSION=100.1.1+up1.11.4 make remove && CHART=rancher-istio VERSION=100.1.2+up1.11.4 make remove`. Save this as it's own commit with the message "make remove"
- Rerun `make charts` and save it as the final commit

In the end of the day, there should be exactly 4 commits:
- bump jaeger to 1.31 (packages/rancher-istio/rancher-tracing changes)
- bump istio to 1.11.7 (packages/rancher-istio/rancher-istio changes AND release.yaml changes)
- make remove (charts/, assets/, index.yaml should be modified with removals only)
- make charts (charts/, assets/, index.yaml should be modified with additions only)

Reply:
@aiyengar2 Thanks for reviewing. I made the changes as you mentioned above. Can you please review it again and let me know if it looks good?